src/patternfly/components/Menu/menu.scss

```diff
@@ -202,33 +202,29 @@
     margin-bottom: var(--pf-c-menu--c-divider--MarginBottom);
   }
 
-  &.pf-m-flyout {
-    overflow: visible;
-
-    .pf-c-menu {
-      position: absolute;
-      top: var(--pf-c-menu--m-flyout__menu--Top);
-      right: auto;
-      left: var(--pf-c-menu--m-flyout__menu--Left);
-
-      &.pf-m-top {
-        top: auto;
-        bottom: var(--pf-c-menu--m-flyout__menu--m-top--Bottom);
-      }
+  // nested flyout menu
+  .pf-c-menu.pf-m-flyout,
+  &.pf-m-flyout .pf-c-menu {
```

Comment:
Can't be written as above, the as that would apply to unnested menus. Lmk if this works.

Reply:
oh I think it would be https://www.sassmeister.com/gist/7b8731fd4020af2215d69b67a0953a35?token=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&scope=gist,read:user

```
.pf-c-menu {
    & &.pf-m-flyout,
    &.pf-m-flyout & {
        color: red;
    }
}
```

doesn't matter though, it doesn't buy us anything and is easier to read the way you've written it.